### ISSUE TITLE
Add Maltese Translation

### ISSUE THREAD

**Issue by stefanbuttigieg** [jp-um/coronatest#8]
We need to start work on the Maltese Translations.

**Comment by jp-um** [comment]
Do you know someone with good Maltese lang. skills who can help?

**Comment by stefanbuttigieg** [comment]
Yes and I will prepare a list that he/she can translate. Will see who I can rope in. Hoping that @hugoamuscat will be able to help us out

**Comment by jp-um** [comment]
you just need to give them the file: `coronatest/survey/src/assets/i18n/en_US.json`

**Comment by stefanbuttigieg** [comment]
File has been given to translators, awaiting translation files

**Comment by stefanbuttigieg** [comment]
Translation of previous JSON file done. Now fine-tuning this translation to include the latest outcomes
